.prow.yaml

```diff
@@ -135,7 +135,7 @@ presubmits:
             - "-c"
             - |
               set -eE; cd "$(git rev-parse --show-toplevel)/e2e/terraform"; trap 'terraform destroy -target module.gcp-ubuntu-focal -auto-approve' EXIT;
-              terraform init && timeout 75m terraform apply -target module.gcp-ubuntu-focal -var="pkg_version=v1.0.1" -auto-approve
+              terraform init && timeout 90m terraform apply -target module.gcp-ubuntu-focal -var="fail_fast=true" -auto-approve
```

Comment:
@electrocucaracha I recommend increasing, even more, the time, I think 120m should be good as we discussed.